### PR TITLE
[ADD] New module <project_task_code>, moved from avanzosc/odoo-addons

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

**Comment by qtheuret** [comment]
LGTM

File: project_task_code/__init__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from . import models
+from openerp import SUPERUSER_ID
+
+
+def create_code_equal_to_id(cr):
+    cr.execute('ALTER TABLE project_task '
```
Comment (pedrobaeza) [inline project_task_code/__init__.py:16]: Comment why is this needed in a docstring

**Comment by pedrobaeza** [comment]
:+1: with a little remark

**Comment by oihane** [comment]
Updated with docstrings...

**Comment by pedrobaeza** [comment]
Thanks! :+1:

**Comment by anajuaristi** [comment]
:+1: RTM from my side

**Comment by oihane** [comment]
@pedrobaeza there are three :+1: if you add @qtheuret LGTM as :+1:

**Comment by pedrobaeza** [comment]
Merging
